gqlpycgen/client.py

```diff
@@ -67,20 +68,40 @@ def __init__(self, uri, accessToken=None, client_credentials=None, timeout=DEFAU
 
     @exponential_retry
     def execute(self, query, variables=None, files=None, timeout=DEFAULT_HTTP_REQUEST_TIMEOUT):
+        overall_start = time.time()
+        logger.debug('Client execute request body JSON:\n{}'.format(query))
```

Comment:
if you change this to:
`logger.debug('Client execute request body JSON:\n%s', query)`

then changing the log level to something that excludes debug will prevent the format string from using unnecessary cycles. Doing format directly will always do the format.